services/telemetry/telemetry.go

```diff
@@ -1075,42 +1075,42 @@ func (ts *TelemetryService) trackElasticsearch() {
 func (ts *TelemetryService) trackGroups() {
 	groupCount, err := ts.dbStore.Group().GroupCount()
 	if err != nil {
-		mlog.Error(err.Error())
+		mlog.Debug("Could not get group_count", mlog.Err(err))
```

Comment:
I think errors for telemetries probably do not interest the system administrator.